srcs/juloo.keyboard2/KeyModifier.java

```diff
@@ -60,6 +60,9 @@ public static KeyValue modify(KeyValue k, KeyValue.Modifier mod)
       case SLASH: return apply_map_char(k, map_char_slash);
       case BAR: return apply_map_char(k, map_char_bar);
       case ARROW_RIGHT: return apply_combining(k, "\u20D7");
+      case DOT_BELOW: return apply_dead_char(k, '\u0323');
+      case HORN: return apply_map_char(k, map_char_horn);
+      case HOOK_ABOVE: return apply_dead_char(k, '\u0309');
```

Comment:
`DOT_BELOW` and `HOOK_ABOVE` don't work.
It seems that these characters use combining diacritics ? In this case, you should use `apply_combining`.

Reply:
no, I think I need to write a map char for this. Currently, I can type all vietnamese characters, but a little bit janky (e.g: ạ vs ạ is different)

Reply:
I'll add map char for those two accent soon

Reply:
If use `apply_combining`, the other accents should be use it too and the whole code is only for Vietnamese, which is a big deal.

Reply:
Do you mean that Vietnamese uses combining diacritics for breve, aigu, grave, etc.. ?

Reply:
yes
ex: `a á à ả ã ạ`

Reply:
In this case, you can simply use `apply_combining` to implement `DOT_BELOW` and `HOOK_ABOVE`.

For the other accents, you'll need to define new versions of them (eg. `AIGU_COMBINING`, etc..) that use the same symbol but are implemented using `apply_combining`.

Reply:
The combining is janky, I added map char and it now work as I (a Vietnamese guy) expected

Reply:
if combining, the accent and character before are two characters, meanwhile almost Vietnamese typing engine didn't use that, they use an actual character

Reply:
The keyboard is perfect for me now. If someone has problem about the Vietnamese layout and accent when using UnexpectedKeyboard, just mention me

Reply:
That works for me.